docs/user/requirements.rst

```diff
@@ -21,18 +20,19 @@ The following packages are needed in order to run RocketPy:
 - Numpy >= 1.0
 - Scipy >= 1.0
 - Matplotlib >= 3.0
-- netCDF4 >= 1.4
+- netCDF4 >= 1.4, < 1.6 for Python 3.7+, netCDF4 >= 1.6.2 for Python 3.11
```

Comment:
is this code line already working properly?

Reply:
This is documentation. It shouldnbe working. Are you getting errors when building the docs?